src/components/webviews/jsInteractions/jsCozyInjection.js

```diff
@@ -1,19 +1,22 @@
 import { Platform } from 'react-native'
 
 import { version } from '../../../../package.json'
-import { statusBarHeight, getNavbarHeight } from '../../../libs/dimensions'
+import { getDimensions } from '../../../libs/dimensions'
 
 const immersiveRoutes = ['home']
 
-const makeMetadata = routeName =>
-  JSON.stringify({
+const makeMetadata = routeName => {
+  const dimensions = getDimensions()
+
+  return JSON.stringify({
     immersive: immersiveRoutes.includes(routeName),
-    navbarHeight: getNavbarHeight(),
+    navbarHeight: dimensions.navbarHeight,
     platform: Platform,
     routeName,
-    statusBarHeight,
+    statusBarHeight: dimensions.statusBarHeight,
     version
   })
+}

```

Comment:
Why not: 
```javascript
const makeMetadata = routeName => {
  const { navbarHeight, statusBarHeight } = getDimensions()

  return JSON.stringify({
    immersive: immersiveRoutes.includes(routeName),
    navbarHeight,
    platform: Platform,
    routeName,
    statusBarHeight,
    version
  })
}
```

Reply:
Fixed: https://github.com/cozy/cozy-flagship-app/compare/5c41f093aaeed8e479d4e6a945b45820d73d7c45..c37e2460728d435323d3615b315a651f7ce4f18f